includes/settings/class-wcpdf-settings-callbacks.php

```diff
@@ -714,6 +717,23 @@ public function validate( $input ) {
 		// Return the array processing any additional functions filtered by this action.
 		return apply_filters( 'wpo_wcpdf_validate_input', $output, $input );
 	}
+
+	/**
+	 * Appends language at the end of the setting provided, in case the setting is translatable
+	 * and it does not have a language set.
+	 *
+	 * @param string $setting Settings field that needs a language.
+	 * @param array  $args Setting arguments.
+	 *
+	 * @return void
```

Comment:
Maybe `string`?

Reply:
Ah yes, I forgot to change that! Thanks.